packages/docsify-server-renderer/index.js

```diff
@@ -1,13 +1,13 @@
 import { readFileSync } from 'fs';
 import { resolve, basename } from 'path';
 import resolvePathname from 'resolve-pathname';
+import fetch from 'node-fetch';
```

Comment:
I think this should be reverted to the previous state. There is no point in rearranging imports if this is the only changes in the file

Reply:
This is a result of the automated husky lint process, which modifies the files. The `git commit` step should not leave modifications in the working tree.